config/prod/JKG-s3.yaml

```diff
@@ -0,0 +1,39 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template_path: remote/s3-bucket.yaml
+stack_name: JKG-Bucket
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "jake.gockley@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username, required if AllowWriteBucket=true
+  SynapseUserName: 'jgockley'
+  # (Optional) Allow accounts, groups, and users to access bucket.
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:iam::563295687221:user/jake.gockley@sagebase.org'
+  
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  #EncryptBucket: 'false'
+  # (Optional) 'Enabled' to enable bucket versioning, default is 'Suspended'
+  #BucketVersioning: 'Enabled'
+  # (Optional) 'Enabled' to enable bucket data life cycle rule, default is 'Disabled'
+  #EnableDataLifeCycle: 'Enabled'
+  # (Optional) S3 bucket objects will transition into this storage class: GLACIER(default), STANDARD_IA, ONEZONE_IA
+  #LifecycleDataStorageClass: 'STANDARD_IA'
+  # (Optional) Number of days until S3 objects are moved to the LifecycleDataStorageClass, default is 30
+  #LifecycleDataTransition: '90'
+  # (Optional) Number of days (from creation) when objects are deleted from S3 and LifecycleDataStorageClass, default is 365000
+  #LifecycleDataExpiration: '1825'
+hooks:
+  before_create:
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/s3-bucket.yaml --create-dirs -o templates/remote/s3-bucket.yaml"
+  before_update:
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/s3-bucket.yaml --create-dirs -o templates/remote/s3-bucket.yaml"
+  after_create:
+    - !synapse_bucket_notify {{stack_group_config.aws_account_name}} {{stack_group_config.aws_account_email}}
```

Comment:
sorry, the example PR was slightly out of date.  we have updated.  please replace `!synapse_bucket_notify` with `!s3_notify`